backend/src/db.js

```diff
@@ -62,6 +62,17 @@ db.setupModels = async () => {
 
     db.Class.hasMany(db.Course, {
       foreignKey: 'classId',
+      as: 'courses',
```

Comment:
het is wat vaag, m aar volgens mij kloppen deze niet. Volgens mij moet de 'as' relatief zijn aan de foreignKey, dus in dit geval as: "class"

goed om dit even te testen.